frontend/src/modules/member/config/filters/activityType/config.ts

```diff
@@ -14,14 +18,14 @@ const activityType: CustomFilterConfig = {
   component: ActivityTypeFilter,
   options: {
   },
-  queryUrlParser: queryUrlParserByType[FilterConfigType.SELECT],
-  itemLabelRenderer(value: SelectFilterValue, options: SelectFilterOptions, data: any): string {
-    return itemLabelRendererByType[FilterConfigType.SELECT]('Activity type', value, data, null, {
+  queryUrlParser: queryUrlParserByType[FilterConfigType.MULTISELECT],
+  itemLabelRenderer(value: MultiSelectFilterValue, options: MultiSelectFilterOptions, data: any): string {
+    return itemLabelRendererByType[FilterConfigType.MULTISELECT]('Activity type', value, data, null, {
       addGroupLabel: true,
     });
   },
-  apiFilterRenderer(value: SelectFilterValue): any[] {
-    return apiFilterRendererByType[FilterConfigType.SELECT]('activityTypes', value);
+  apiFilterRenderer(value: MultiSelectFilterValue): any[] {
+    return apiFilterRendererByType[FilterConfigType.MULTISELECT]('activityTypes', value);
```

Comment:
The filter is being applied with an `and` operator. However, there will never be multiple activity types in one activity. Can you apply a similar logic that you did with `type` in organizations, where you use the `or` operator?